css/main.css

```diff
@@ -0,0 +1,59 @@
+body{
+    margin : 0;
+}
+.navigation ul li{
+    float: left;
```

Comment:
Most modern developers only use floats as a last resort. There are better ways to lay things out. ex grid or flexbox

- Flexbox: https://css-tricks.com/snippets/css/a-guide-to-flexbox/
- Grid: https://css-tricks.com/snippets/css/complete-guide-grid/

Reply:
i will try with display flex